src/modules/Printer.rb

```diff
@@ -664,11 +644,9 @@ def DetermineDriverOptions(queue_name)
         end
       end
       return false if "" == queue_name
-      commandline = Ops.add(
-        @determine_printer_driver_options_commandline,
-        queue_name
-      )
-      if !Printerlib.ExecuteBashCommand(commandline)
+      if !Printerlib.ExecuteBashCommand(
+           @determine_printer_driver_options_commandline + queue_name.shellescape
```

Comment:
I guess this is the place where
https://github.com/yast/yast-printer/pull/23#issuecomment-475285328
happens.

Perhaps there are other places where a change from
```
Ops.add( this, that)
```
to
```
this + that
```
causes a missing blank?
But I know nothing about Ruby coding details.

Reply:
No, it doesn't do anything fancy like that; it's just an overly complex way to write

```
this + that
```

But I had overlooked the other nested Ops.add() that had added a trailing blank to that command line.

Reply:
But commit 957ac4d24e098e4b0e91487332fd28a356599715 should fix this now.